kotlin/render-multilayer-symbols/src/main/java/com/esri/arcgisruntime/sample/rendermultilayersymbols/MainActivity.kt

```diff
@@ -0,0 +1,595 @@
+                Graphic(
+                    Point(130.0, 50.0, SpatialReferences.getWgs84()),
+                    getTextSymbol("Multilayer\nComplex Symbols")
+                )
+            )
+        )
+    }
+
+    /**
+     * @return the TextSymbol with the [text] to be displayed on the map.
+     */
+    private fun getTextSymbol(text: String): TextSymbol {
+        val textSymbol = TextSymbol(
+            10F,
+            text,
+            Color.BLACK,
+            TextSymbol.HorizontalAlignment.CENTER,
+            TextSymbol.VerticalAlignment.MIDDLE
+        )
+        // give the text symbol a white background
+        textSymbol.backgroundColor = Color.WHITE
+        return textSymbol
+    }
+
+    /**
+     * Create picture marker symbols from online URI or local cache.
+     */
+    private fun addImageGraphics() {
+        // URI of image to display
+        val bluePinImageURI =
+            "https://sampleserver6.arcgisonline.com/arcgis/rest/services/Recreation/FeatureServer/0/images/e82f744ebb069bb35b234b3fea46deae/blue_pin.png"
+        // load the PictureMarkerSymbolLayer using the image URI
+        val pictureMarkerFromUri = PictureMarkerSymbolLayer(bluePinImageURI)
+        pictureMarkerFromUri.addDoneLoadingListener {
+            if (pictureMarkerFromUri.loadStatus == LoadStatus.LOADED) {
+                // add loaded layer to the map
+                addGraphicFromPictureMarkerSymbolLayer(pictureMarkerFromUri, 0.0)
+            } else {
+                showError("Picture marker symbol layer failed to load from URI: ${pictureMarkerFromUri.loadError.message}")
+            }
+        }
+        pictureMarkerFromUri.loadAsync()
+
+        val localCachePath = cacheDir.toString() + File.separator + getString(R.string.blue_pin)
+        // load blue pin from assets into cache directory
+        copyFileFromAssetsToCache(localCachePath)
+        // load the PictureMarkerSymbolLayer using the image using local cache
+        val pictureMarkerFromCache = PictureMarkerSymbolLayer(localCachePath)
+        pictureMarkerFromCache.addDoneLoadingListener {
+            if(pictureMarkerFromCache.loadStatus == LoadStatus.LOADED){
+                // add loaded layer to the map
+                addGraphicFromPictureMarkerSymbolLayer(pictureMarkerFromCache, 20.0)
+            } else {
+                showError("Picture marker symbol layer failed to load from cache: ${pictureMarkerFromCache.loadError.message}")
+            }
+        }
+        pictureMarkerFromCache.loadAsync()
+    }
+
+    /**
+     * Copy the given file from the app's assets folder to the app's cache directory.
+     * @param localCachePath as String.
+     */
+    private fun copyFileFromAssetsToCache(localCachePath: String) {
+        val assetManager = applicationContext.assets
+        val file = File(localCachePath)
+        if (!file.exists()) {
+            try {
+                val `in` = assetManager.open(getString(R.string.blue_pin))
```

Comment:
I wonder if fileIn and fileOut might make this easier to read rather than escaping a keyword